FSHD_CLI/README.md

```diff
@@ -0,0 +1,88 @@
+
+
+
+# Haplotype Predictor
+
+This Python script allows you to predict genotypes from Short Sequence Length Polymorphisms (SSLPs) values for Facioscapulohumeral muscular dystrophy (FSHD) analysis. It requires a selection of SSLPs and the region from where the sample is derived.
```

Comment:
region=population :wink:
I suggest: "(...) the population to which the individual belongs, from where the sample is derived."

Reply:
Fixed in aaf66dc.